src/helium_api_wrapper/challenges.py

```diff
@@ -107,14 +107,16 @@ def load_challenge_data(
         challenges = challenges
 
     for challenge in challenges:
-        witnesses = __sort_witnesses(challenge.witnesses, load_type=load_type)
-        challengee = get_hotspot_by_address(address=challenge.challengee)
-
+        if challenge.witnesses is not None:
+            witnesses = __sort_witnesses(challenge.witnesses, load_type=load_type)
+        if challenge.challengee is not None:
+            challengee = get_hotspot_by_address(address=challenge.challengee)
+        
         for witness in witnesses:
```

Comment:
witnesses gets defined in an if clause, meaning that it's possible that in line 115 witnesses is undefined. Is that the expected behaviour?